### PR TITLE
Use bot in harvester

### PR DESCRIPTION
**Pull request by futuretea**
https://github.com/harvester/harvester/issues/1819

update code and ci to use this bot in harvester

### CODE REVIEW

**Comment by futuretea** [comment]
1. @guangbochen  templates/pre-merge.md need to be updated 

2. Refer to https://github.com/longhorn/bot/pull/1
The repository contains two scripts, github-bot and github-runner-authorizer, the github-bot is the one we need. The github-runner-authorizer needs to be accompanied by github-runner, but there is no introduction or code for github-runner in this repository.

3. There is no manifest yamls (deploy, service) in this repository

File: .drone.yml [hunk 1]
```diff
@@ -138,25 +80,17 @@ steps:
...
     - push
 depends_on:
-- longhorn-bot
-- longhorn-github-runner-authorizer
-
-
-volumes:
-- name: socket
-  host:
-    path: /var/run/docker.sock
```
Comment (guangbochen) [inline .drone.yml:162]: the volume is needed for docker build.

Comment (futuretea) [reply .drone.yml:162]: The harvester-bot pipeline already contains this volume. No docker build in the deploy-harvester-bot-images pipeline

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**